scripts/globals/additional_effects.lua

```diff
@@ -120,10 +120,11 @@ xi.additionalEffect.attack = function(attacker, defender, baseAttackDamage, item
         ABSORB_STATUS = 10,
         SELF_BUFF = 11,
         DEATH = 12,
+        DAMAGE_SPECIFIC_MOB = 13,
```

Comment:
Sorry but we already said we didn't want to do this exact thing on a previous pr, tho at the time our understanding of how the knife worked was off. I recently did a kill of brig. blade to be sure I understood, and some additional tests on the knife itself.

These are my notes from that: 

> tested hitting it with the dagger then unequipping. the dagger does not "remove" its immunity. It becomes the only way to damage it once it gets to 1% hp Melee hit did zero, add effect killed it. 
>
> add effect damage does proc on other evil weapon type mobs NEARBY but not elsewhere in other zones, and for far less dmg. 43 vs a mystic weapon and it did not proc vs a goobue gardener at all nor an evil weapon in rollanberry
>
> it does work from the offhand - all my hits were offhand hits, mainhand was a Shijo.
> so we need latent effect for that proc. vs sub-family only. weapons in ve lugunon gonna need diff subfamily ID than the ones in the rest of the game.
> 
> we can on-hit listener, and after dealing 0 dmg from the melee hit remove immortal status and remove his damage taken reduction so the en-damage kills him.

I don't think I finished hooking up that latent. Either main family or subfamily wasn't hooked up yet. its an easy thing to do I just forgot about it is all.

Reply:
pardon me, misspoke subfamily isn't a thing, its family and super family:

```sql
  `familyID` smallint(4) unsigned NOT NULL,
  `family` tinytext,
  `superFamilyID` smallint(4) unsigned NOT NULL DEFAULT 0,
  `superFamily` tinytext,
  `ecosystemID` tinyint(2) unsigned NOT NULL DEFAULT 0,
  `ecosystem` tinytext,
```
```
-- "family, superFamily, ecosystem" relationship: Korrigan, Madragora, Plantoid
-- Nothing is enforced so it is possible to use completely unrelated values
```

Reply:
the family latent this would need is hooked up, it was superfamily I never finished, which I'll get in soon (workin on it right now) so I can put _bano del sol_ to rest

Reply:
@TeoTwawki Did you ever finish the "superfamily"?
I'm looking to fix the Bano Del Sol proc rate bug.